src/OrleansSQLUtils/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +22,7 @@
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
```

Comment:
cannot check it for you since main solution doesn't compile, for the reason in comments below